ruddock/modules/rotation/routes.py

```diff
@@ -11,13 +11,18 @@
 @login_required(Permissions.ROTATION)
 def show_portal():
   dinner_list = helpers.DINNERS
+  buckets = helpers.BUCKETS
   return flask.render_template('portal.html',
-    dinner_list=dinner_list)
+    dinner_list=dinner_list, buckets=buckets)
 
+@blueprint.route('/dinner', defaults={'dinner_id': None})
```

Comment:
no dinner id just returns all prefrosh. maybe i should rename this route `/directory` and make the dinner id an optional GET parameter?